src/main/java/io/javalin/core/JavalinConfig.java

```diff
@@ -83,6 +93,34 @@
     }
     // @formatter:on
 
+    /**
+     * Register a new plugin.
+     */
+    public void registerPlugin(@NotNull Plugin plugin) {
+        if (isPluginRegistered(plugin.getClass())) {
+            throw new PluginAlreadyRegisteredException(plugin.getClass());
+        }
+        inner.plugins.put(plugin.getClass(), plugin);
+    }
+
+    /**
+     * Returns true if the plugin is registered.
+     */
+    public boolean isPluginRegistered(@NotNull Class<? extends Plugin> pluginClass) {
+        return inner.plugins.containsKey(pluginClass);
+    }
+
+    /**
+     * Get a registered plugin or null if it is not defined
```

Comment:
```suggestion
     * Get a registered plugin (or null if it is not defined)
```

Reply:
I just realised that this section is not true anymore as I throw an error if the plugin was not found.